docs/pages/blog/mui-material-v5-3.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Material v5.3 — Theming upgrade
```

Comment:
Interesting, I wonder if we should have a blog post for each product, or one for the whole MUI Core line? Like, we could imagine:
```suggestion
title: MUI Core v5.3.0
```

Reply:
@oliviertassinari  Should I keep the `— Theming upgrade` at the end? I think it makes the title more attractive.

Reply:
I would go with MUI Material.

Reply:
From my perspective, there is dissonance. For example, take the tweet:

<img width="613" alt="Screenshot 2022-01-20 at 11 50 34" src="https://user-images.githubusercontent.com/3165635/150324878-2558f77e-2363-436e-86e6-03a209599c26.png">

We have on one side a changelog for MUI Core, but on the other hand, a blog post for `@mui/material`. I don't think that it really works. What I would suggest is one of these two:

- We make the blog post for MUI Core v5.3.0, we cover the key changes
- We make the blog post specifically for the new theming API, the same way @alexfauquette focuses on one specific change of the MUI X v5.3.0 release #30598.

Reply:
Alright, I think going in the same direction as @alexfauquette is nice! Starting with feature highlight and then mention the release is better in term of SEO (I guess) and the title is more attractive.

Reply:
👌